docs/index.md

```diff
@@ -0,0 +1,49 @@
+# YamlConfiger --- AMQ configurator
+
+This tool can generate a set of configuration files mainly needed for
+AMQ Broker, but it is not limited to only generating files for one product.
+
+YamlConfiger has a user facing Command Line Tool for quick and easy command line usage.
+Furthermore, it is possible to use its API in your python code.
+
+## Documentation
+
+Formatted documentation can be viewed at [rh-messaging-qe.github.io/YamlConfiger/index.html](https://rh-messaging-qe.github.io/YamlConfiger/index.html).
+
+## Contributing
+
+If you find a bug or room for improvement, submit either a ticket or PR.
+
+## Contributors
+
+_Alphabetically ordered_
+
+* Michal Tóth <mtoth@redhat.com>
```

Comment:
Maybe this list should be changed.... Added Dominik, Howard, Jiri already...
AUTHORS.md?

Reply:
Either AUTHORS.md, or we could add CODEOWNERS to the file tree https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners, and https://github.com/rh-messaging-qe/YamlConfiger/graphs/contributors to docs

Reply:
doc updated in comments, CODEOWNERS is left for further consideration